samples/README.md

```diff
@@ -0,0 +1,27 @@
+VIC Sample Data
+===============
+
+This directory includes complete sample datasets to run VIC. Sample datasets that are currently included in the VIC source code repository:
+
+## Image Driver
+1.  Stehekin - 10-days of forcings
+
+## Classic Driver
+
+1.  Stehekin - 10-days of forcings
+
+## Summary of Datasets:
+1.  Stehekin: This is a distributed datasets that covers the entire Stehekin River Basin, located in Washington State, USA.
+
+## Getting the Datasets:
+The VIC sample datasets are in a "git submodule", to download the datasets to this directory, simply run the following command:
+
+```
+./get_sample_data.bash
+```
+
+You may also download the sample data to another location on your computer using the following command:
```

Comment:
Maybe we should be a bit more specific here, e.g. `You may also download the sample data to another location on your computer by specifying the directory in which you would like to add the data:` 

`git clone https://github.com/jhamman/VIC_sample_data.git  path/folder_name` 

This would be helpful for someone less familiar with using Git.